packages/core/src/dialects/cockroachdb/query-interface.ts

```diff
@@ -0,0 +1,31 @@
+import type { QueryRawOptions } from '../../index';
+import { PostgresQueryInterface } from '../postgres/query-interface';
+
+export class CockroachDbQueryInterface extends PostgresQueryInterface {
+  async dropSchema(schema: string, options: QueryRawOptions) {
+    if (schema === 'crdb_internal') {
+      return;
+    }
+
+    await super.dropSchema(schema, options);
+  }
+
+  async removeConstraint(tableName: string, constraintName: string, options: QueryRawOptions): Promise<any> {
+    try {
+      await super.removeConstraint(tableName, constraintName, options);
+    } catch (error: any) {
+      if (error.message.includes('use DROP INDEX CASCADE instead')) {
```

Comment:
if you're going to use cascade automatically, might as well add it as part of the query from the start instead of running two queries